src/Avalonia.FuncUI.TreeDataGrid/Avalonia.FuncUI.TreeDataGrid.fsproj

```diff
@@ -0,0 +1,43 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <Version>$(FuncUIVersion)</Version>
+    <Authors>JaggerJo</Authors>
+    <Product>Avalonia.FuncUI.TreeDataGrid</Product>
+    <PackageId>Avalonia.FuncUI.TreeDataGrid</PackageId>
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
+    <PackageProjectUrl>https://github.com/fsprojects/Avalonia.FuncUI</PackageProjectUrl>
```

Comment:
On a related note, should some of these common properties (url, author, etc) go in a common props file?